manifests/init.pp

```diff
@@ -28,6 +28,7 @@
   $epel_gpgcheck                                       = $epel::params::epel_gpgcheck,
   $epel_repo_gpgcheck                                  = $epel::params::epel_repo_gpgcheck,
   $epel_metalink                                       = $epel::params::epel_metalink,
+  Boolean $epel_metalink_enabled                       = true,
```

Comment:
please document the new parameters with puppet-strings style in the header of the file